pkg/logs/config/config.go

```diff
@@ -24,6 +24,8 @@ const (
 	httpEndpointPrefix = "agent-http-intake.logs."
 )
 
+const defaultCompressionLevel = 6
```

Comment:
What does 6 stand for ? Good compression with okay speed ? Can we add a comment please ? Also why not using -1 like here https://golang.org/pkg/compress/flate/#pkg-constants ?

Reply:
Also you probably want to set the default value [here](https://github.com/DataDog/datadog-agent/blob/master/pkg/config/config.go#L396) so that this attribute can be used as environment variable

Reply:
6 is the actual default level: https://github.com/golang/go/blob/master/src/compress/flate/deflate.go#L597-L599.

I don't want to put `-1` because it is not explicit on which level it is mapped.

I will add a comment to explain that value.